pydsa/dijkstrashortestpath.py

```diff
@@ -0,0 +1,46 @@
+import heapq
+import sys
+
+
+
```

Comment:
sure. will get it done. It seems I missed checking pep8 standard in few lines. :+1: